ExcelMapper/StringAttribute.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Ganss.Excel
+{
+    /// <summary>
+    /// Attribute that specifies how string type mapping to be handled.
+    /// </summary>
+    /// <seealso cref="System.Attribute" />
+    [AttributeUsage(AttributeTargets.Property, Inherited = true, AllowMultiple = false)]
+    public sealed class StringAttribute : Attribute
```

Comment:
This attribute should have a different name, perhaps `FormulaResultAttribute`. Also it should have a default constructor so you can say just `[FormulaResult]` to map the formula result instead of the formula.